app/components/kickout-modal.hbs

```diff
@@ -0,0 +1,18 @@
+<div data-test-kickout-modal class='kickout-modal'>
```

Comment:
I think instead of making modals for specific cases the base modal should just take `heading`, `confirmButtonText`, `onConfirm` etc as props and be reusable